app/assets/stylesheets/location.css

```diff
@@ -0,0 +1,16 @@
+.leaflet-marker-icon {
+  background: url("https://raw.githubusercontent.com/pointhi/leaflet-color-markers/master/img/marker-icon-black.png") ; 
```

Comment:
@nstjean, is there a reason behind adding this code? This is currently turning all marker icons' background to black.

Reply:
@crisner Looks like I just moved that code from rich.html.rb into a stylesheet. So feel free to remove it if it's causing issues!

Reply:
Aha, yes i think this will overwrite LEL css also. 
@nstjean , ideally we should make this a functionality in PL.editor to change marker color by taking input from user. 
But for now let's move this css back as it may cause issues in future 😄

Reply:
Interesting. I don't actually see it changing the marker color in /post/ - I'm wondering if we can just remove it altogether. @crisner where are you seeing the color change?

Reply:
Here:https://publiclab.org/map
![publiclab org_map](https://user-images.githubusercontent.com/29401459/72276459-f217fd80-3655-11ea-9307-77253611953f.png)

And here: https://publiclab.org/wiki/unearthing-pvd
![publiclab org_wiki_unearthing-pvd](https://user-images.githubusercontent.com/29401459/72276537-268bb980-3656-11ea-8ab2-4d76504bce2e.png)

I guess wherever we have maps.

Reply:
Ahhh, now I see where it is!
Fix in PR #7222

Reply:
Awesome!